<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>progress_bar_fixed_in_statusbar</class>
 <widget class="QWidget" name="progress_bar_fixed_in_statusbar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>322</width>
    <height>24</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>3</number>
     </property>
     <item>
      <widget class="QLabel" name="label_record_displayed_from_tota_record">
       <property name="minimumSize">
        <size>
         <width>160</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>160</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QProgressBar" name="progressBar">
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>8</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>120</width>
         <height>8</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QProgressBar {
text-align: top;
border-bottom-right-radius: 0px;
border-bottom-left-radius: 0px;
background:#cdcdcd;

}
QProgressBar::chunk {
background: #15A9ED;
border-bottom-right-radius: 0px;
border-bottom-left-radius: 0px;

}</string>
       </property>
       <property name="minimum">
        <number>0</number>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="value">
        <number>30</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_close">
       <property name="minimumSize">
        <size>
         <width>8</width>
         <height>8</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>8</width>
         <height>8</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">
QPuhButton:flat{border: none; background-color: transparent;}
QPuhButton{background:transparent;}
QPushButton:checked{ background-color: transparent; border: none;  }
QPushButton:!checked{ background-color: transparent; border: none;  } 
QPushButton:clicked{ background-color: transparent;  border: none;}               
QPushButton:pressed { background-color: transparent; border: none; }

</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>12</width>
         <height>5</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_status_message">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
